src/aap_eda/wsapi/messages.py

```diff
@@ -85,6 +85,17 @@ class ControllerInfo(BaseModel):
     ssl_verify: str
 
 
+class VaultPassword(BaseModel):
+    type: str = "VaultPassword"
```

Comment:
- What is the purpose of a type str? I think it should be a enum: https://docs.pydantic.dev/2.3/usage/types/enums/
- Do we really need a type? What are the other types? because the field "password" is a mandatory string.

Reply:
@Alex-Izquierdo Every message sent on the WebSocket Channel has a type they are all listed [here](https://github.com/ansible/eda-server/blob/eed9db3923c1c2b1674fdddd335142beb55d4768/src/aap_eda/wsapi/messages.py#L21)